chainweb.cabal

```
@@ -325,8 +325,7 @@ library
         , attoparsec >= 0.13
         , base >= 4.12 && < 5
         , base16-bytestring >= 0.1
-        , base64-bytestring >= 1.0 && < 1.1
-            -- due to change in error message that is inlined in pact
+        , base64 >= 0.5 && < 0.6
```

Is this known to break with base64 >=0.6? Otherwise please remove, since we don't manually track "soft" upper bounds.

```suggestion
        , base64 >= 0.5
```